[2021-04-21 20:15:33,512] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.sleep 2021-04-19T08:10:00+00:00 [queued]>
[2021-04-21 20:15:33,561] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: docker_sample.sleep 2021-04-19T08:10:00+00:00 [queued]>
[2021-04-21 20:15:33,563] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:15:33,565] {taskinstance.py:1043} INFO - Starting attempt 1 of 4
[2021-04-21 20:15:33,566] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-04-21 20:15:33,589] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): sleep> on 2021-04-19T08:10:00+00:00
[2021-04-21 20:15:33,606] {standard_task_runner.py:52} INFO - Started process 596 to run task
[2021-04-21 20:15:33,614] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'docker_sample', 'sleep', '2021-04-19T08:10:00+00:00', '--job-id', '253', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dockerOperator.py', '--cfg-path', '/tmp/tmppn2dl09s', '--error-file', '/tmp/tmpze2xm464']
[2021-04-21 20:15:33,625] {standard_task_runner.py:77} INFO - Job 253: Subtask sleep
[2021-04-21 20:15:33,777] {logging_mixin.py:104} INFO - Running <TaskInstance: docker_sample.sleep 2021-04-19T08:10:00+00:00 [running]> on host 5af2b1cab26e
[2021-04-21 20:15:33,936] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=docker_sample
AIRFLOW_CTX_TASK_ID=sleep
AIRFLOW_CTX_EXECUTION_DATE=2021-04-19T08:10:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-04-19T08:10:00+00:00
[2021-04-21 20:15:33,937] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-04-21 20:15:33,939] {bash.py:158} INFO - Running command: sleep 5
[2021-04-21 20:15:33,961] {bash.py:169} INFO - Output:
[2021-04-21 20:15:38,965] {bash.py:177} INFO - Command exited with return code 0
[2021-04-21 20:15:39,038] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=docker_sample, task_id=sleep, execution_date=20210419T081000, start_date=20210421T201533, end_date=20210421T201539
[2021-04-21 20:15:39,072] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-04-21 20:15:39,087] {local_task_job.py:146} INFO - Task exited with return code 0
